iac-extensions/kubernetes/src/main/java/org/sonar/iac/helm/utils/HelmFilesystemUtils.java

```diff
@@ -111,4 +108,31 @@ private static String resolveToInputFile(Path helmDirectoryPath, InputFile addit
   public static String normalizeToUnixPathSeparator(String filename) {
     return filename.replace('\\', '/');
   }
+
+  public static String normalizeToRuntimePathSeparator(String filename) {
+    return filename.replace('\\', File.separatorChar)
```

Comment:
Additional comment: Also, maybe it would be easier to just use unix separators in the whole `HelmProcessor.readCompressedFile` method. Windows is able to read these paths as well.

Reply:
I wanted to be sure if calling `File.getCanonicalPath()` works on each system. 
If `/` works everywhere then I'm ok to use it, but this requires additional tests.